source/core/ut_suite_manager.pkb

```diff
@@ -441,7 +442,7 @@ create or replace package body ut_suite_manager is
 
           l_item_name := l_item.name;
           --l_item_name := regexp_substr(l_item_name,'[A-Za-z0-9$#_]+$'); -- temporary fix. seems like suite have suitepath in object_name
```

Comment:
Can we have the commented code removed? This temp filx is there since quite some time ;)